packages/steps/src/cli/cli.ts

```diff
@@ -6,32 +6,38 @@ import { v4 as uuidv4 } from 'uuid';
 import { BuildConfigParser } from '../BuildConfigParser.js';
 import { BuildStepContext } from '../BuildStepContext.js';
 import { BuildWorkflowError } from '../errors.js';
+import { BuildPlatform } from '../BuildPlatform.js';
 
 const logger = createLogger({
   name: 'steps-cli',
   level: 'info',
 });
 
-async function runAsync(configPath: string, workingDirectory: string): Promise<void> {
+async function runAsync(
+  configPath: string,
+  workingDirectory: string,
+  platform: BuildPlatform
+): Promise<void> {
   const fakeBuildId = uuidv4();
-  const ctx = new BuildStepContext(fakeBuildId, logger, false, workingDirectory);
+  const ctx = new BuildStepContext(fakeBuildId, logger, false, platform, workingDirectory);
   const parser = new BuildConfigParser(ctx, { configPath });
   const workflow = await parser.parseAsync();
   await workflow.executeAsync();
 }
 
 const relativeConfigPath = process.argv[2];
 const relativeWorkingDirectoryPath = process.argv[3];
+const platform: BuildPlatform = (process.argv[4] ?? process.platform) as BuildPlatform;
 
-if (!relativeConfigPath && !relativeWorkingDirectoryPath) {
-  console.error('Usage: yarn cli config.yml path/to/working/directory');
+if (!relativeConfigPath || !relativeWorkingDirectoryPath || !platform) {
```

Comment:
I think this is never gonna be falsy.
```suggestion
if (!relativeConfigPath || !relativeWorkingDirectoryPath) {
```